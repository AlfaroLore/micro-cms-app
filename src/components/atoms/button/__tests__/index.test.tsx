import { render } from '@testing-library/react';
import Button from '..';

describe('Button', () => {
  it('Renders Button', () => {
    const component = render(<Button />);

    expect(component).toBeTruthy();
    expect(component.asFragment()).toMatchSnapshot();
  });
});
