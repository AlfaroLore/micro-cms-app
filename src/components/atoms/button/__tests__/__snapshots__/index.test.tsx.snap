// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Button > Renders Button 1`] = `
<DocumentFragment>
  <button
    class="sc-gTRrQi jocjXQ"
    type="button"
  />
</DocumentFragment>
`;
